extensionsGallery-insider.json

```diff
@@ -2848,6 +2848,63 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "59",
```

Comment:
Please update the TotalCount value at the bottom as well for both this and extensionsGallery.json